xml2rfc/writers/pdf.py

```diff
@@ -61,6 +62,14 @@ def __init__(self, xmlrfc, quiet=None, options=default_options, date=None):
         else:
             ftlogger.setLevel(logging.ERROR)
 
+    def flatten_unicode_spans(self, html):
+        # This is a fix for bug in WeasyPrint that doesn't handle RTL unicode
+        # content correctly.
+        # See #873 & Kozea/WeasyPrint#1711
+        return re.sub(r'<span class="unicode">(?P<unicode_content>.*)</span>',
```

Comment:
Shouldn't this be a `.*?` so it doesn't bleed over into additional spans?
Maybe add a test case with `<u>a</u>b<u>c</u>` as well.

Reply:
Yes, you are correct. It's greedy by default. I'll fix that and add an unit test.

Reply:
I've updated the commit.

Reply:
LGTM now.